### PR TITLE
Remove taskID from built-in run type

### PR DESCRIPTION
**Pull request by colinking**
When executing a built-in, we previously returned a `Run` type which included a `taskID: string`. That field was never set, so this change updates the types to clarify that. Executing a task (via `execute()`) will return a `TaskRun` while executing a built-in (via any of the built-in functions, like `sql.query()`) will return a `BuiltInRun`.

cc @leeweisberger 

Resolves TSK-221

### CODE REVIEW

**Comment by colinking** [comment]
Current dependencies on/for this PR:
* colin-09-06-Directly_export_new_methods
  * **PR #127** <a href="https://app.graphite.dev/github/pr/airplanedev/node-sdk/127" target="_blank">Graphite</a> 
    * **PR #128** <a href="https://app.graphite.dev/github/pr/airplanedev/node-sdk/128" target="_blank">Graphite</a> 
      * **PR #129** <a href="https://app.graphite.dev/github/pr/airplanedev/node-sdk/129" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/airplanedev/node-sdk/129?utm_source=stack-comment).

File: packages/airplane/internal/execute.ts [hunk 1]
```diff
@@ -31,7 +31,13 @@ export const executeWithResources = async <Output = unknown>(
     throw new RunTerminationError(run);
   }
 
-  return run;
+  if (!run.taskID) {
```
Comment (calufornia) [inline packages/airplane/internal/execute.ts:34]: It looks like all the builtins above call `executeWithResources`, but from my understanding of the PR description, the run object that's returned for builtins doesn't have `taskID` set, so wouldn't validation fail here?

File: packages/airplane/internal/execute.ts [hunk 2]
```diff
@@ -31,7 +31,13 @@ export const executeWithResources = async <Output = unknown>(
...
+    // `execute` is intended for executing tasks, not built-ins.
+    // This is a sanity check before we cast to a `Run` type that includes a taskID.
+    throw new Error(`Run does not have a taskID`);
+  }
+
+  return run as Run<typeof params, Output>;
```
[the remark this answers is not in the document]
Comment (colinking) [reply packages/airplane/internal/execute.ts:40]: Good catch! I had switched the naming here halfway through and missed this 😄

File: packages/airplane/internal/api/types.ts [hunk 3]
```diff
@@ -8,14 +8,29 @@ export type SessionStatus =
...
-  output: O;
+  output: TOutput;
+};
+
+// Run is a run created from a task.
+export type TaskRun<TParamValues extends ParamValues = ParamValues, TOutput = unknown> = Run<
+  TParamValues,
+  TOutput
+> & {
+  taskID: string;
```
[the remark this answers is not in the document]
Comment (colinking) [reply packages/airplane/internal/api/types.ts:25]: Yep!

File: packages/airplane/internal/execute.ts (hunk 1 above)
Comment (colinking) [reply packages/airplane/internal/execute.ts]: Fixed! Good catch 🙌🏻

**Review by calufornia — APPROVED** [review]

**Pull request merged**